.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
+on:
+#  push:
+#    branches:
+#      - main
+  pull_request:
+  workflow_dispatch:
+
+# we might need this to allow queued workflows to interrupt previous runs
+#concurrency:
+#  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+#  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: corretto
+          cache: sbt
+          java-version: 8
+
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: eu-west-1
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+
+      - name: Clean, compile and test
+        run: sbt clean compile test assembly
+
+      - name: Upload to riff-raff
+        uses: guardian/actions-riff-raff@v2
+        with:
+          app: live-app-versions
+          configPath: riff-raff.yaml
+          contentDirectories: |
+            live-app-versions:
+              - target/scala-2.12/live-app-versions.jar
+            live-app-versions-cfn:
+              - cfn.yaml
```

Comment:
We may like a follow-on ticket to migrate this project to cdk...

Reply:
absolutely - great idea

Reply:
By the way, we already have a ticket for this